pandaserver/userinterface/Client.py

```diff
@@ -1,68 +1,33 @@
 """
 client methods
-
 """
 
-import getpass
 import gzip
 import json
 import os
 import pickle
-import re
 import socket
 import sys
 import tempfile
-import uuid
-from urllib.parse import urlencode
 
+import requests
 from pandacommon.pandautils.net_utils import replace_hostname_in_url_randomly
 
-from pandaserver.srvcore.CoreUtils import commands_get_status_output
-
-# configuration
-try:
-    baseURL = os.environ["PANDA_URL"]
-except Exception:
-    baseURL = "http://pandaserver.cern.ch:25080/server/panda"
-try:
-    baseURLSSL = os.environ["PANDA_URL_SSL"]
-except Exception:
-    baseURLSSL = "https://pandaserver.cern.ch:25443/server/panda"
-
+# PanDA server configuration
+baseURL = os.environ.get("PANDA_URL", "http://pandaserver.cern.ch:25080/server/panda")
+baseURLSSL = os.environ.get("PANDA_URL_SSL", "https://pandaserver.cern.ch:25443/server/panda")
 
 # exit code
 EC_Failed = 255
 
 
-# panda server URLs
-if "PANDA_URL_MAP" in os.environ:
```

Comment:
https://github.com/tmaeno/panda-server/blob/master/templates/sysconfig/panda_server_env.systemd.rpmnew.template#L33

This kinds of lines can be removed